lib/rubocop/cop/lint/redundant_type_conversion.rb

```diff
@@ -52,6 +52,13 @@ module Lint
       #   {}
       #   Set.new
       #
+      #   # bad - chaining to a type constructor
+      #   Integer(var).to_i
+      #
+      #   # good - chaining to a type constructor with exceptions suppressed
+      #   # in this case, `Integer()` could return `nil`
+      #   Integer(var, exception: false).to_i
```

Comment:
It may be unclear why these incompatible cases are listed as `bad` and `good` side by side.

```console
$ ruby -e "Integer('a', exception: false).to_i" # => 0
$ ruby -e "Integer('a').to_i"
<internal:kernel>:288:in 'Kernel#Integer': invalid value for Integer(): "a" (ArgumentError)
        from -e:1:in '<main>'
```